lib/TabTracker.js

```diff
@@ -164,8 +173,6 @@ TabTracker.prototype = {
     // already ended, likely with an 'unfocus' unload reason.
     if (this.isActivityStreamsURL(tab.url) && tabs.activeTab.id === tab.id) {
       if (!this._tabData.url) {
-        this._tabData.url = tab.url;
-        this._tabData.tab_id = tab.id;
```

Comment:
I removed these two since `logActivate()` will be called a few lines down and it already sets them